index.html

```diff
@@ -1 +1,11 @@
-
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Welcome to potionfy</title>
+  </head>
+  <body>
+    <h1>Welcome to potionfy! Enjoy your stay.</h1>
```

Comment:
Could you remove the 'Enjoy your stay.' string and make the 'Welcome to potionfy!' string red?